LayoutTests/accessibility/element-reflection.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<script src="../resources/js-test.js"></script>
+<script src="../resources/accessibility-helper.js"></script>
+<label id="label1">First label</label>
+<input id="input1"></input>
+<label>Second label</label>
+<input id="input2"></input>
+<label id="label3">Third label</label>
+<x-input-idref></x-input-idref>
+
+<script>
+  class XInputIDREF extends HTMLElement {
+    constructor() {
+      super();
+      this.attachShadow({ mode: "open" });
+      let input = document.createElement("input");
+      input.id = "innerinput";
+      this.shadowRoot.appendChild(input);
+      input.ariaLabelledByElements = [label3];
+    }
+  }
+  customElements.define("x-input-idref", XInputIDREF);
+
+  input1.ariaLabelledByElements = [label1];
```

Comment:
Please  ariaActiveDescendantElement, ariaControlsElements, ariaDescribedByElements, ariaDetailsElements, ariaErrorMessageElement, ariaFlowToElements, ariaLabelledByElements, and ariaOwnsElements.

Reply:
I've added tests for all of this but I have some some problems testing custom elements... I need to investigate if that's a WebKitGTK issue or also happens on Mac platforms. Uploaded new patch to check EWS though tests are still failing locally (running things manually on minibrowser shows the proper relationships on the a11y panel in the inspector).